tests/test_tutorials.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+cd ../tutorials
+g++ -std=c++11 -march=native -o3 lattice_base_generation_tutorial.cpp -lfplll -lmpfr -lgmp -o test_base
```

Comment:
I think those calls should be unified in a loop, avoiding repeating the same lines several times